src/components/TipControl.vue

```diff
@@ -154,14 +172,13 @@ export default {
     async sendTip() {
       const amount = util.aeToAtoms(this.value);
       this.showLoading = true;
-      await aeternity.tip(this.tip.url, this.title, amount)
+      await aeternity.tip(`${window.location.origin}/#/tip/${this.tip.id}`, this.message, amount)
```

Comment:
can we first consider to use url without the domain. I'm very concerned about leaking random internal urls stored in the blockchain.

Reply:
also it makes this untestable

Reply:
URL without the domain is not URL, rather a path.